docs/mine-hnt/validators/mainnet/deployment-guide.mdx

```diff
@@ -461,6 +461,17 @@ miner peer book -s
 
 You're looking for `listen_addrs`. If you don't have at least one, your Validator hasn't settled on how to tell other peers to reach it. Often this can take 15-30 mins, sometimes longer.
 
+### Copying your swarm_key
+Your validator's swarm_key is located in the validator_data/miner directory
+
+In order to copy the swarm_key locally to your personal computer, you'll need to use scp (outside of/instead of an ssh session).  Replace the user name and IP address (`root@192.0.2.1`) in the example below with the actual user name and IP address of your docker container instance.
```

Comment:
```suggestion
In order to copy the `swarm_key` locally to your personal computer, you'll need to use scp (outside of/instead of an ssh session).  Replace the user name and IP address (`root@192.0.2.1`) in the example below with the actual user name and IP address of your docker container instance.
```